RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: minor
+
+This release blocks installation of Hypothesis on Python 3.3, which
+:PEP:`reached its end of life date on 2017-09-29 <398>`
+
+This should not be of interest to anyone but downstream maintainers and
+possibly packaging geeks - if you are affected, migrate to a secure version
```

Comment:
I'd prefer not to use the term "geeks" like this. Although I'm sure it's not intended that way, it comes off as pejorative.

Reply:
No problem - best to avoid any misunderstanding with an international audience.